### DISCUSSION TITLE
How to use a cluster to build Sentinel mode

### DISCUSSION THREAD

**Discussion by githubzhangfei** [dragonflydb/dragonfly#2904] [Q&A]
How should I set up the Sentinel mode? Is there any relevant documentation? Or is using Redis as a sentinel deployment feasible? Should high availability be included in the plan as soon as possible? I don't know if the author has any relevant scheduling arrangements

**Comment by kostasrim** [comment]
Hi @githubzhangfei ? Sentinel should work, pls take a look at https://www.dragonflydb.io/docs/migrating-to-dragonfly/from-redis-instance/sentinel-promotion#introduction

**Reply by githubzhangfei** [reply]
There is another question I would like to ask you, whether Dragonfly has such a configuration, I do not want the server to actively close the link, whether he has such a configuration. My demand for concurrency is so high that I don't want to close the link after creating it in Java. Not sure if dragonfly supports it
